test/src/e2e.dynval/dv.double-vote.test.ts

```diff
@@ -138,7 +139,95 @@ async function afterBanCheck(sdk: SDK, bestBlockNumber: number) {
 
 describe("Report Double Vote", function() {
     const promiseExpect = new PromiseExpect();
-    const termSeconds = 20;
+
+    async function getPossibleAuthorsByString(
+        sdk: SDK,
+        blockNumber: number
+    ): Promise<string[]> {
+        return (await stake.getPossibleAuthors(sdk, blockNumber))!.map(
+            platfromAddr => platfromAddr.toString()
+        );
+    }
+
+    describe("Ban from the validator state", async function() {
+        const nodes = withNodes(this, {
+            promiseExpect,
+            validators: allDynValidators.map((signer, index) => ({
+                signer,
+                delegation: 5_000,
+                deposit: 10_000_000 - index // tie-breaker
+            }))
+        });
+
+        it("alice should be banned from the validators", async function() {
+            const secsPerblock = 3;
+            this.slow(secsPerblock * 7 * 1000);
+            this.timeout(secsPerblock * 14 * 1000);
+
+            const checkingNode = nodes[1];
+            const blockNumber = await checkingNode.sdk.rpc.chain.getBestBlockNumber();
+            const termMedata = await stake.getTermMetadata(
+                checkingNode.sdk,
+                blockNumber
+            );
+            const currentTermInitialBlockNumber =
+                termMedata!.lastTermFinishedBlockNumber + 1;
+            expect(termMedata!.currentTermId).to.be.equals(1);
+            await beforeBanCheck(
+                checkingNode.sdk,
+                currentTermInitialBlockNumber
+            );
+            await checkingNode.waitBlockNumber(
+                currentTermInitialBlockNumber + 2
+            );
+            const aliceAddr = alice.platformAddress.toString();
+
+            const aliceIdxInBestBlock = (await getPossibleAuthorsByString(
+                checkingNode.sdk,
+                currentTermInitialBlockNumber
+            )).indexOf(aliceAddr);
+
+            const alicePrivkey = alice.privateKey;
+            const consensusMessage1 = encodableMessage({
+                height: currentTermInitialBlockNumber,
+                view: 0,
+                step: "precommit",
+                blockHash: new H256(
+                    "730f75dafd73e047b86acb2dbd74e75dcb93272fa084a9082848f2341aa1abb6"
```

Comment:
Is this just a random string?

Reply:
Yes, I just generated random hash.

Reply:
I think it'd be better to change this to more unrealistic value (ex. `0000....00001`) to prevent collision.